src/contexts/DataContext.js

```diff
@@ -0,0 +1,17 @@
+import React, { createContext } from 'react';
+import { useFirebase } from '../hooks/useFirebase';
+
+const DataContext = createContext();
+
+const DataContextProvider = ({ children }) => {
```

Comment:
I'm super impressed by the React skills that I'm seeing from the group so far!

React Context is super awesome, however, I'm not sure if we're taking full advantage of its power/identifying the optimal use-case. Typically, we use Context to create global state that can be easily shared by different components throughout the app. In this case, there actually is no state being held in the Context, so I don't know if it's needed. Instead, we could just use the `useFirebase` hook directly in the components. In general, I try to avoid premature optimization.

With that being said, we could make this Context have some immediate value now if we wanted to have something like loading state held in it. That way we could ensure that all of the components are listening to the same global loading state. Alternatively, we could also hold loading state in a hook, but then it would be state that is unique to that particular instantiation of the hook. Part of the fun of programming is that there are so many options!

Adding and consuming loading state would be above and beyond the acceptance criteria and thus, slightly out of scope, but I did just want to throw that idea out there in case you were interested.  :-)

Reply:
Update to may comment - I see that the react-firebase-hooks already include the loading state. So, I think that the Context may be unnecessary and we could just consumer your super awesome hook directly in the component. Totally up to you though, because this does work as is.

Reply:
Thank you for your feedback @MikeBLambert! we discussed it with the rest of the team and agreed with you that it was kind of redundant to keep the context on. We have made the changes so that the actions required for firebase are called directly from the useFirebase custom hook.